webapi/Models/Storage/ChatParticipant.cs

```diff
@@ -26,6 +27,12 @@ public class ChatParticipant : IStorageEntity
     /// </summary>
     public string ChatId { get; set; }
 
+    /// <summary>
+    /// The partition key for the source.
+    /// </summary>
+    [JsonIgnore]
+    public string Partition => this.ChatId;
```

Comment:
Should participants partitioned by user id? A user can be multiple participants in separate chats. We use participants to locate the chats a user belongs to.

Reply:
All the models/entities are "chat-centric".  When querying this container entity, look-up occurs by chatid or chatid & userid.